wiki/Performance_Points/en.md

```diff
@@ -1,156 +1,171 @@
-        -   Drain Time = Song length - Breaktime - Intro/outro.
--   **@ difficulty** + [Mods](Game_Modifiers "wikilink")
-    -   "+ [Mods](Game_Modifiers "wikilink")" excluded list: [Sudden Death](Sudden_Death "wikilink")/[Perfect](Perfect "wikilink"), [Relax](Relax "wikilink"), [Auto Pilot](Auto_Pilot "wikilink"), [Auto](Auto "wikilink")/[Cinema](Cinema "wikilink").
+### How does aim, speed, accuracy and strain combine to produce a pp score?
 
-Weightage system
-----------------
+**All four elements are considered in concert to determine how difficult a map is overall, and also how a particular score compares to what is considered the 'optimal' play for that beatmap.**
 
-<img src="Pp bp.jpg" title=" Click the picture for pp formula usage example." alt=" Click the picture for pp formula usage example." width="275" />
+The algorithm for performance points varies significantly depending on gamemode.
 
-For transparency on how your final pp is calculated, pp gained from the beatmap is given along with weight (% of eligible pp to be gained for final pp). Only your highest pp score's beatmap gives you full pp. Other scores' pp values will be multiplied with decreasing percentage. The percentage is always rounded to the next whole number for displaying, but for calculations it is not. For example a score can be weighted by 0.48% despite the display saying "0%". If *n* is the amount of scores giving more pp than a given score, then the score's weight is **0.95**^*n* .
+While the exact numbers are well beyond the scope of this article, certain gamemodes place greater precedence on certain statistics due to their individual mechanics.
 
-It follows, that your total pp is computed as follows. Let *PP* contain each score's pp value. *PP*\[i\] denotes the *i*th score's pp value, sorted decreasingly, where *i* goes from 1 to *n*, and *n* is the amount of scores you have.
+### What is the 'weightage system' and how does it affect my scoring?
 
-**Total pp** = *PP*\[1\] \* **0.95**^0 + *PP*\[2\] \* **0.95**^1 + *PP*\[3\] \* **0.95**^2 + ... + *PP*\[*n*\] \* **0.95**^(*n*-1)
+**The weightage system refers to the fact that all of your scores are weighted against each other in terms of overall performance.**
 
-[Click for an example of the formula usage (scroll down to see the example).](/wiki/Pp_bp.jpg)
+This means that only your highest pp score will award the full amount of pp for that particular performance. Other scores with lower overall pp values will be weighted with a gradually decreasing percentage.
 
-There is also bonus pp based on the number of ranked maps you have a score on. The bonus is:
+To provide an example of this with a mathematical formula:
 
-416.6667 (1- 0.9994^Number\_of\_scores).
+Let _PP_ represent each score's individual pp value. _PP\[i\]_ denotes the _i_th score's pp value, sorted descending, where _i_ ranges from 1 to _n_, and _n_ is defined as the amount of scores you have.
```

Comment:
just pointing this out since it appeared on GitHub (though, I have no clue if this also happens live)
![chrome_2017-08-02_22-05-23](https://user-images.githubusercontent.com/14264992/28904198-c89ef0b4-77ce-11e7-9dd8-f301c650e09c.png)
I assume GitHub's Markdown doesn't like it when you mid-italic parts of a word or something 🤷‍♂️